sdk/src/main/java/com/talkable/sdk/Talkable.java

```diff
@@ -181,12 +181,16 @@ private static void setNativeFeatures(Context context) {
             isMessengerInstalled = MessengerUtils.hasMessengerInstalled(context);
         }
 
+        Intent sendNativeMailIntent = new Intent(Intent.ACTION_SENDTO, Uri.parse("mailto:"));
+        Boolean canSendNativeMail = !context.getPackageManager().queryIntentActivities(sendNativeMailIntent, 0).isEmpty();
```

Comment:
Please move it under `if (context != null)` and set default value to `false` as `isSmsAvailable & isMessengerInstalled ` do

Reply:
`null` is a bane of Java